src/lib.rs

```diff
@@ -637,6 +650,23 @@ mod tests {
         );
     }
 
+    #[test]
+    fn xor_name_from_content() {
+        let alpha_1 = XorName::from_content(&["abcdefg".as_bytes(), "hijk".as_bytes()]);
```

Comment:
Minor: instead of `as_bytes()`, you can use the byte string literal: `b"abcdefg"`